torch_geometric/distributed/dist_context.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+from typing import Optional
+
+
+class DistRole(Enum):
+    WORKER = 1  # for worker mode
+
+
+class DistContext(object):
```

Comment:
I wonder if we can make this a `@dataclass`? Seems like it is just a struct with some properties.

Reply:
done